libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -93,11 +93,12 @@ func getCgroupRoot() (string, error) {
 }
 
 type cgroupData struct {
-	root   string
-	parent string
-	name   string
-	config *configs.Cgroup
-	pid    int
+	root           string
+	parent         string
+	name           string
+	config         *configs.Cgroup
+	pid            int
+	externalCgroup bool
```

Comment:
We shouldn't require this right? If there is a path passed in then we use that path.